automation/check-patch.e2e-k8s.sh

```diff
@@ -17,7 +17,7 @@ main() {
     make cluster-up
     trap teardown EXIT SIGINT SIGTERM SIGSTOP
     make cluster-sync
-    make E2E_TEST_ARGS="-ginkgo.noColor --junit-output=$ARTIFACTS/junit.functest.xml" functest
+    make E2E_TEST_ARGS="-ginkgo.noColor --ginkgo.junit-report=$ARTIFACTS/junit.functest.xml" functest
```

Comment:
you need to use --output-dir $ARTIFACTS, and to remove the dir from the junit report please

remove also all the teardown that manipulate those files

note that spyglass doesn't print errors good until there will be a bump of it

Reply:
Let's see if [it](https://github.com/k8snetworkplumbingwg/kubemacpool/pull/362/commits/114302fa9e8229b186b1fa847b3df721417b908b) works

Reply:
Thanks
Q: Dont we still need `--junit-output` in addition ?
unless the default file name that is used is good

Reply:
it isn't, you're right

Reply:
https://onsi.github.io/ginkgo/MIGRATING_TO_V2#generating-machine-readable-reports
yes
you can see other repos like CNAO i think, where it is used when bumped to ginkgo 2.0

if you want, lets postphone it to a new PR

Reply:
we do.. so.. I don't understand why it doesn't work. checking..